deployments/common/image/common-scripts/fix-certs

```diff
@@ -1,10 +1,26 @@
-#! /bin/bash -eu
```

Comment:
Are failures OK here?

Reply:
No.   I just moved the -eu down to line 7 on the right so that it executes after system RC files have already been sourced.  Could easily be moved back,  the main benefit of that switch is for scripts in tools... which when executed on OS-X cause bogus warnings/errors since system RC files with undefined variables are sourced after the shebang -eu is already in effect.  Moving -eu down OS-X stops bogus complaints so I changed this out of habit...  on Linux its really irrelevant I think.

Reply:
Ah, I see.  That makes sense.